docker/Dockerfile

```diff
@@ -1,69 +1,11 @@
-FROM rocker/r-ver:3.6.0
-
-RUN apt-get update && apt-get -y install \
-        libcurl4-openssl-dev \
-        libgdal-dev \
-        libprotobuf-dev \
-        libssh-dev \
-        libudunits2-dev \
-        libv8-dev \
-        libxml2-dev \
-        protobuf-compiler \
-        texinfo \
-        texlive \
-        texlive-fonts-extra \
-        zlib1g-dev
+FROM mrcide/naomi_base:latest
```

Comment:
I should have noticed this before sorry. I think that docker tags are almost always dashes - PR for the actual image https://github.com/mrc-ide/naomi.docker/pull/2 and then this can become naomi-base